_data/simple-icons.json

```diff
@@ -10457,7 +10463,7 @@
         {
             "title": "Zig",
             "hex": "F7A41D",
-            "source": "https://github.com/ziglang/logo",
+            "source": "https://github.com/ziglang/logo/blob/master/zig-mark.svg",
```

Comment:
Should be https://github.com/ziglang/logo/blob/608770bf7303613c18a8c3faf284516fa31072f0/zig-mark.svg